app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -6,6 +6,10 @@ def call(toolbar_name)
     build_toolbar(toolbar_name)
   end
 
+  def call_by_class(toolbar_class)
+    build(toolbar_class)
+  end
```

Comment:
So the new method does exactly the same as the old one except it has a different name. We surely don't need that.